src/content/guides/code-splitting.md

```diff
@@ -280,6 +280,8 @@ __src/index.js__
   });
 ```
 
+T> It is possible to provide a [dynamic expression](/api/module-methods/#dynamic-expressions-in-import) to `import()` when you might need to import specific module based on a computed variable later.
```

Comment:
Last part sounds weird, What you think about:
> based on a variable computed later.

Reply:
we are trying to say: "use `import()` to import something later (maybe based on a variable that you will compute)"